doc/manual/manual.tex

```diff
@@ -2868,6 +2868,38 @@ \subsubsection{Adjusting solver tolerances} At the heart of every time step
 point where the output quantities start to change significantly. This is the
 point where you will want to stop.
 
+\subsubsection{Adjusting solver preconditioner tolerances} To solve the Stokes 
+equations it is necessary to lower the condition number of the
+Stokes matrix by preconditioning  it. In \aspect{} a right preconditioner $Y^{-1} = 
+\begin{pmatrix}
+\widetilde{A^{-1}} & -\widetilde{A^{-1}}B^{T}\widetilde{S^{-1}} \\
+0 & \widetilde{S^{-1}}
+\end{pmatrix}$ is used to precondition the system, where $\widetilde{A^{-1}}$ is 
+the approximate inverse of the A block and $\widetilde{S^{-1}}$ is the approximate 
+inverse of the Schur complement matrix. Matrix $\widetilde{A^{-1}}$ and 
+$\widetilde{S^{-1}}$ are calculated through a CG solve, which requires a tolerance 
+to be set. In comparison with the solver tolerances of the previous section, these 
+parameters are relatively safe to use, since they only change the preconditioner, 
+but can speed up or slow down the solving of the Stokes system considerably. 
+
+In practice $\widetilde{A^{-1}}$ takes by far the most time to compute, but is 
+also very important in conditioning the system. The computation of 
+$\widetilde{A^{-1}}$ is controlled by the parameter \texttt{Linear solver A 
+block tolerance} which has a default value of $1e-2$. Setting this tolerance 
+to a less strict value will result in more outer iterations, since the 
+preconditioner is not as good, but the amount of time to compute 
+$\widetilde{A^{-1}}$ can drop significantly resulting in a reduced total solve 
+time. The cookbook crustal deformation (Section 
+\ref{sec:cookbooks-crustal-deformation}) for example can be computed much faster 
+by setting the \texttt{Linear solver A block tolerance} to $5e-1$. The 
+calculation of $\widetilde{S^{-1}}$ is usually much faster and the 
+conditioning of the system is less sensitive to the parameter \texttt{Linear 
+solver S block tolerance}, but for some problems it might be worth it to 
```

Comment:
I think this was actually grammatically correct already. Either "it is worth it to investigate" or maybe better "it is worthwhile to investigate" (or "it is worthy of investigation"; isn't the English language wonderful?)